one-ui/src/test/java/com/marklogic/hub/oneui/TestHelper.java

```diff
@@ -23,7 +23,7 @@
     public String dataHubDeveloperPassword;
 
     @Autowired
-    private HubConfigSession hubConfig;
+    public HubConfigSession hubConfig;
```

Comment:
I think we can avoid using public class variables. You can autowire HubConfigSession or use a getter

Reply:
@srinathgit I just changed it based on your suggestion.